### DISCUSSION TITLE
Proposal: Add the upload beatmap function in the Laser editor

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#21156] [Ideas] [answered]
I made a beatmap in Osu! Laser and I wanted to upload it to the servers. I had to import the beatmap into the stable Osu! version and upload it there. Could someone please make a UI for doing that?

**Comment by peppy** [comment, marked as answer]
One day this will be possible. That day is not today.

**Reply by IGN-Styly** [reply]
is that day the 1st of january 2024?

**Reply by yesseruser** [reply]
> is that day the 1st of january 2024?

I don't think so. The editor is still very buggy from my experience.

**Reply by bdach** [reply]
the "bugginess" of the editor (???) has nothing to do with the fact that the systems to allow beatmap submission are not yet in place

**Reply by RoughlyUnderscore** [reply]
Is this being worked on or should we not expect it yet?

**Reply by bdach** [reply]
I'm not working on it yet until the basic usability concerns with the editor itself are resolved.

**Reply by lebaaru** [reply]
it's still not here ):

**Reply by bdach** [reply]
work on this is in progress. please exercise patience.

**Reply by yesseruser** [reply]
I'm glad to see it is being worked on in the end. Thank you.
